client/src/styles/LandingPage.css

```diff
@@ -0,0 +1,71 @@
+
+@import url('https://fonts.googleapis.com/css?family=Raleway');
+@import url('https://fonts.googleapis.com/css?family=Oswald');
+
+html, body {
+    margin: 0;
+    padding: 0;
+    height: 100%;
+    width: 100%;
+}
+
+.intro {
+    height: 100vh;
+    width: 100vw;
+    margin: auto;
+    background: url("../components/images/lotus.jpg") no-repeat 50% 50%;
+    display: table;
```

Comment:
Reactstrap has easy compatibility with flexbox, and I've seen that tables can do their own thing sometimes. Would it be difficult to convert this to use flexbox instead?

Reply:
XD For me it might be a difficult, but I don't mind looking into it if we need it! I sent you a demo of the landing page, tho, and its pretty responsive so maybe we don't need flexbox?

Reply:
I updated the landing page to flexbox! It was easier than I thought it would be!